packages/past-reminders/components/PastRemindersPosts/index.jsx

```diff
@@ -120,6 +122,12 @@ const PastRemindersPosts = ({
     );
   }
 
+  if (hasCampaignsFeature) {
```

Comment:
Talking to Hamish the other day I discovered we usually put the if conditions inside the useEffect. 
```suggestion
useEffect(() => {
  if(hasCampaignsFeature) {
    fetchCampaigns();
  }
}, []);
```

Reply:
Ah great one! I'll move it 😊